clients/cobol-lsp-vscode-extension/src/services/LanguageClientService.ts

```diff
@@ -109,7 +108,7 @@ export class LanguageClientService {
             params: ConfigurationParams,
             token: vscode.CancellationToken,
             next: ConfigurationRequest.HandlerSignature) => {
-            return await this.middleware.handleConfigurationRequest(params, token, next);
+            return next(params, token);
```

Comment:
If I don't miss anything, I think you don't need this middleware configuration at all now.